x-pack/plugins/actions/server/builtin_action_types/jira/index.ts

```diff
@@ -109,21 +108,22 @@ async function executor(
     throw new Error(errorMessage);
   }
 
+  if (subAction === 'getIncident') {
+    const getIncidentParams = subActionParams as ExecutorSubActionGetIncidentParams;
+    const res = await api.getIncident({
+      externalService,
+      params: getIncidentParams,
+    });
+    if (res != null) {
```

Comment:
I am not sure if this check is needed as if the incident is not there Jira will return 404 and axios will thrown an error.

Reply:
I did this because the type returned can be `null`. without it i get a type error